code/game/jobs/job_controller.dm

```diff
@@ -429,9 +429,9 @@ var/global/datum/controller/occupations/job_master
 					return H.Robotize()
 				if("AI")
 					return H
-				if("Captain")
+				if("Station Administrator")
 					var/sound/announce_sound = (ticker.current_state <= GAME_STATE_SETTING_UP)? null : sound('sound/misc/boatswain.ogg', volume=20)
-					captain_announcement.Announce("All hands, Captain [H.real_name] on deck!", new_sound=announce_sound)
+					captain_announcement.Announce("All hands, Station Administrator [H.real_name] on deck!", new_sound=announce_sound)

```

Comment:
Could probably use an alt-title checker, not really necessary, but would be nice.